### ISSUE TITLE
Unhandled switch case FT_PrefAlign after 193d7a6

### ISSUE THREAD

**Issue by citymarina** [swiftlang/llvm-project#13395]
Upstream's 193d7a6 added `FT_PrefAlign` to `MCFragment`'s `enum FragmentType`, leading to this warning:

```
llvm/lib/MCCAS/MCCASObjectV1.cpp:1877:11: note: add missing switch cases
 1877 |   switch (F.getKind()) {
      |           ^
```

`MCFRAGMENT_NODE_REF` will need to be updated in `llvm/include/llvm/MCCAS/MCCASObjectV1.def`.

I can silence the warning with something like
```
MCFRAGMENT_NODE_REF(MCPrefAlignFragment, FT_PrefAlign, mc:pref_align)
```
but I'm not confident to land the change myself. In particular:
* I don't know whether it should be within the `MCFRAGMENT_ENCODED_FRAGMENT_ONLY` section or not
* Also I don't know whether there are other areas that have special handline for `Align` that should also include `PrefAlign`? One example is https://github.com/swiftlang/llvm-project/blob/d76c5bac701c3731be9025d0834b876e7283e564/llvm/lib/MCCAS/MCCASObjectV1.cpp#L1957 .

Are there any active maintainers of MCCAS still around?
